docs/containers/debug-node.md

```diff
@@ -124,10 +115,107 @@ Also note that the `debug` logger will not write logs unless enabled via the `DE
 }
 ```
 
-### Automatically launching the browser to the entry page of the application
+### Configuring when the application is "ready"
+
+The extension determines the application is "ready" to receive HTTP connections when it writes a message of the form `Listening on port <number>` to the debug console, as Express.js does by default.  If the application logs a different
+message, then the `pattern` property of the [`dockerServerReadyAction`]((debug-common.md#dockerServerReadyAction-object-properties)) object of the debug launch configuration should be set to a [JavaScript regular expression](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions) that matches that message. The regular expression should include a capture group that corresponds to the port on which the application is listening.
```

Comment:
"should be set" - rewrite this as "you should set .."

Reply:
Updated.